### PR TITLE
slim down the old site even more, static stats page.

### PR DESCRIPTION
**Pull request by mashehu**

### CODE REVIEW

**Comment by mashehu** [comment]
@nf-core-bot fix linting

File: public_html/index.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?php
+echo "This is the cemetery of what was once the PHP code for the nf-core website. The current site is at nf-co.re.Nothing to see here except redirects to specific pages we haven't ported yet.";
+
+?>
```
Comment (ewels) [inline public_html/index.php:4]:
```suggestion
This is the cemetery of what was once the PHP code for the nf-core website. The current site is at https://nf-co.re - nothing to see here except redirects to specific pages we haven't ported yet.
```

Comment (ewels) [reply public_html/index.php:4]: Don't think you need any PHP here!

**Review by ewels — APPROVED** [review]

**Comment by ewels** [comment]
> @nf-core-bot fix linting

The linting is broken atm I think. But about to switch so who cares.

**Pull request merged**